services/director-v2/src/simcore_service_director_v2/modules/dynamic_sidecar/scheduler/_core/_events_utils.py

```diff
@@ -134,7 +135,7 @@ async def service_remove_containers(
         await sidecars_client.stop_service(
             scheduler_data.endpoint, progress_callback=progress_callback
         )
-    except BaseHttpClientError as e:
+    except (BaseHttpClientError, TaskExceptionError) as e:
```

Comment:
so are we going to get twice the same exception then?

Reply:
And logged at every exception handler ... (in reference to my previous comment about logging int he library)

Reply:
This is the case where the exception is handled. So only the warning is diplayed.
There are other places where the exception will be raised and only once traceback will be shown.

--- 

If the exception was not handled a warning + the client's traceback will be displayed.